js/src/dropdown.js

```diff
@@ -289,6 +289,31 @@ class Dropdown extends BaseComponent {
     return this._element.closest(`.${CLASS_NAME_NAVBAR}`) !== null
   }
 
+  _getOffset() {
+    let { offset } = this._config
+
+    if (!offset) {
+      return [0, 0]
+    }
+
+    if (typeof offset === 'number') {
+      offset = [offset, 0]
+    } else if (typeof offset === 'string') {
+      offset = offset.split(',')
+      if (offset.length === 1) {
+        offset = [offset[0], 0]
+      }
+
+      offset = offset.map(val => Number.parseInt(val, 10))
+    } else if (typeof offset === 'function') {
+      offset = (popper, reference, placement) => {
+        return offset({ popper, reference, placement }, this._element)
+      }
```

Comment:
And Popper passes a single object in the offset function: https://popper.js.org/docs/v2/modifiers/offset/#options
You should use destructuring `({ placement, reference, popper })`

Reply:
To be honest, my intention was to pass one object as the first parameter (like was for offsets in bs4)
so it is my mistake on reading Popper docs.